packages/material/scss/grid/_layout.scss

```diff
@@ -34,6 +34,18 @@ $grid-group-drag-clue-opacity: .9 !default;
             box-shadow: none;
         }
 
+        .k-grid-toolbar {
```

Comment:
this is not needed. It's already loaded from line 19. You need to change `$grid-toolbar-button-spacing` to 0.